### PR TITLE
users index view #139155099

### PR DESCRIPTION
**Pull request by patmbolger**
PT Story: https://www.pivotaltracker.com/story/show/139155099

Show all users (view available to admin)

Changes proposed in this pull request:
1. Add users_controller.rb
2. Add views and translations
3. Add feature test

Screenshots (Optional):
<hr>
screen shot 2017-02-16 at 6 10 10 pm
<hr>
screen shot 2017-02-16 at 6 10 24 pm

Ready for review:
@weedySeaDragon 
@thesuss

### CODE REVIEW

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
Would you please add scenarios that show that:
- a visitor cannot see the users page
- a user cannot see the users page
- a member cannot see the list of users

The should get the 'you cannot access this' error message.

The policy will handle this.  Might also want to put this within the `admin` scope in the routes?  (I'd have to check back into that stuff to be sure that this would be the right approach.)

**Comment by patmbolger** [comment]
@weedySeaDragon - improved the (admittedly weak) test coverage.

**Review by weedySeaDragon — APPROVED** [review]

File: config/locales/sv.yml [hunk 1]
```diff
@@ -378,6 +381,17 @@ sv:
     new_business_category: Skapa ny företagstyp (kategori)
     all_business_categories: Alla kategorier
 
+  users:
+    index:
+      title: Alla Unvändare
```
Comment (thesuss) [inline config/locales/sv.yml]: Alla användare

File: config/locales/sv.yml [hunk 2]
```diff
@@ -378,6 +381,17 @@ sv:
...
+      email: *email
+      created: *created
+      last_login: Senaste inloggning
+      logged_in_count: Inloggningar
+      member: Medlem
+      applications: Tillämpningar
```
Comment (thesuss) [inline config/locales/sv.yml]: Ansökningar

File: config/locales/sv.yml [hunk 3]
```diff
@@ -378,6 +381,17 @@ sv:
     new_business_category: Skapa ny företagstyp (kategori)
     all_business_categories: Alla kategorier
 
+  users:
+    index:
+      title: Alla Unvändare
+      email: *email
+      created: *created
+      last_login: Senaste inloggning
```
Comment (thesuss) [inline config/locales/sv.yml]: Senast inloggad

**Pull request merged**